### ISSUE TITLE
Terraform v0.12x changes not working if TMPDIR not set with trailing slash

### ISSUE THREAD

**Issue by spongebrain** [antonbabenko/pre-commit-terraform#50]
Thanks for the update...however, I received the following error when running:

```$ pre-commit run -a
Check for merge conflicts................................................Passed
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
hookid: terraform_docs

mktemp: failed to create file via template ‘/tmpterraform-docs-XXXXXXXXXX’: Permission denied

```

Looked at code and ran `export TMPDIR=/tmp/`, then:
```
$ pre-commit run -a
Check for merge conflicts................................................Passed
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
hookid: terraform_docs

2019/06/18 11:15:40 At 34:15: nested object expected: LBRACE got: ASSIGN
```
Thanks!

**Comment by BastianM3** [comment]
@spongebrain just ran into the same issue.

**Comment by antonbabenko** [comment]
@spongebrain `mktemp` is magic. :( Please use [v1.16.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.16.0) where this was fixed (see c9ecd72).

**Comment by spongebrain** [comment]
The TMPDIR issue is resolved at v1.16.0.  Thanks much!!!

Insofar as the LBRACE vs ASSIGN issue...we're adopting the new v12 HCL format, and thus don't want to revert the code back to 11 syntax.  Are there plans to address that in a future version?

**Comment by antonbabenko** [comment]
I am not sure that LBRACE vs ASSIGN issue you are talking about has anything to do with terraform-docs.

terraform-docs requires a code to look like Terraform 0.11 to be able to process it. This is what `awk` script is doing in a temporary folder.

You should be able to use Terraform 0.12 and let the `terraform_docs` hook do the magic.
